torchvision/prototype/datasets/_builtin/__init__.py

```diff
@@ -1,22 +1,26 @@
 from .caltech import Caltech101, Caltech256
-from .celeba import CelebA
+
```

Comment:
I only made changes to four datasets to cover all cases, i.e.

- HTTP download (CIFAR)
- GDrive download (Caltech)
- Kaggle download (FER2013)
- Manual download (ImageNet)

The others are commented out to be able to import `torchvision`.